vidarr-server/src/test/java/ca/on/oicr/gsi/vidarr/server/MainIntegrationTest.java

```diff
@@ -492,29 +492,6 @@ public void whenGetWorkflowVersionWithDefinition_thenWorkflowVersionWithDefiniti
             containsString("accessoryFiles"));
   }
 
-  @Test
-  public void whenAddExistingWorkflowVersion_thenAddIsRejected() throws JsonProcessingException {
```

Comment:
This test is deleted because adding the same workflow version should succeed and adding a modified version should fail , which are covered by tests `whenAddDuplicateWorkflowVersion_thenWorkflowVersionIsUnchanged` and `whenAddModifiedWorkflowVersion_thenRequestFails` added below.